### ISSUE TITLE
The pagination does not work

### ISSUE THREAD

**Issue by marchwia** [reyesoft/ngx-jsonapi#9]
Given instance `xService` of service `XService extends Service`,
when I call
```
xService.all(
  {
    page: { number: 3, limit: 10 },
    include: ['y']
  },
  success => {},
  error => {}
);
```

I expect to see a request `GET x?include=y&page[number]=3&page[limit]=10`,
but instead I see `GET x?include=y`.

**Comment by pablorsk** [comment]
Hi @marchwia!

thanks for your report. The problem was solved on #13, released very soon.

**Comment by pablorsk** [comment]
Hi @marchwia, your problem was solved on [0.1.0-rc.2](https://www.npmjs.com/package/ngx-jsonapi) :)
